### PR TITLE
Flutter3 Upgrade + Splash Screen

### PR DESCRIPTION
**Pull request by 4855062503**

### CODE REVIEW

**Comment by djpnewton** [comment]
CI is failling

File: .github/workflows/flutter.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ jobs:
           sed -i -e "s|BUILD_DATE_REPLACE|$(date)|g" lib/config.dart
       - uses: subosito/flutter-action@v1
         with:
-          flutter-version: '2.10.x'
+          flutter-version: '3.0'
```
Comment (djpnewton) [inline .github/workflows/flutter.yml:28]: better to use 3.0.x so it picks up bug fixes

File: lib/config.dart [hunk 2]
```diff
@@ -56,9 +56,9 @@ String server() {
   if (UniversalPlatform.isWeb) {
     var location = html.window.location;
     if (BeServerLocationOverrides.keys.contains(location.hostname))
-      serverUrl = location.origin + '/';
+			String serverUrl = location.origin ?? '';
```
Comment (djpnewton) [inline lib/config.dart:59]: redeclearing `serverUrl` means that BeServerLocationOverrides will no longer work

File: lib/main.dart [hunk 3]
```diff
@@ -52,19 +52,9 @@ class MyApp extends StatelessWidget {
       theme: ThemeData(
           textTheme: ZapTextThemer(Theme.of(context).textTheme),
           primaryTextTheme: ZapTextThemer(Theme.of(context).textTheme),
-          colorScheme: ColorScheme(
```
Comment (djpnewton) [inline lib/main.dart:55]: lets not update the color scheme just yet, (just do the splash

**Review by djpnewton — CHANGES_REQUESTED** [review]
also in general:
*dont* put tabs into files that are *already* using spaces

**Comment by 4855062503** [comment]
@djpnewton

Replaced `lib/config.dart` & `lib/main.dart` with the ones in your master branch but still getting flutter build errors in CI

**Comment by 4855062503** [comment]
Just realized I still have to make the splash page mobile friendly.

**Pull request merged**